src/gpu/nvidia/cudnn_convolution.cpp

```diff
@@ -83,20 +81,22 @@ status_t cudnn_convolution_fwd_t::execute_convolution(
             auto handle = cuda_stream->get_cudnn_handle();
 
             std::vector<void *> args;
-            args.push_back(sc.memory<void *>(ih, x_acc));
-            args.push_back(sc.memory<void *>(ih, weights_acc));
-            args.push_back(sc.memory<void *>(ih, y_acc));
+            args.push_back(arg_src.get_native_pointer(ih, sc));
+            args.push_back(arg_weights.get_native_pointer(ih, sc));
+            args.push_back(arg_dst.get_native_pointer(ih, sc));
             args.push_back(
-                    with_bias ? sc.memory<void *>(ih, *bias_acc) : nullptr);
-            args.push_back(with_scratchpad ? sc.memory<void *>(ih, *scratch_acc)
-                                           : nullptr);
+                    with_bias ? arg_bias.get_native_pointer(ih, sc) : nullptr);
```

Comment:
`get_native_pointer(...)` returns `nullptr` when `sycl_memory_arg_t` is empty so we can just do `args.push_back(arg_bias.get_native_pointer(ih))`.

Reply:
Done.